File: Week3_Challenge/Reverse.java
package Week3_Challenge;
import java.util.*;
import java.io.*;
public class Reverse {
        public static String FirstReverse(String str) {
            // Create a StringBuilder object and pass the input string as an argument
            StringBuilder reversedStr = new StringBuilder(str);

            // Use the reverse() method to reverse the string
            reversedStr.reverse();

            // Convert the StringBuilder object back to a string
            String reversed = reversedStr.toString();

            return reversed;
        }

        public static void main(String[] args) {
            // keep this function call here
            Scanner input = new Scanner(System.in);
            System.out.print(FirstReverse(input.nextLine()));
        }

}

File: Week3_Challenge/Reverse.js
function FirstReverse(str) {
  // Split the string into an array of characters
  var charArray = str.split('');
  // Reverse the array of characters
  var reversedArray = charArray.reverse();
  // Join the reversed array of characters back into a string
  var reversedStr = reversedArray.join('');

  return reversedStr;

}

// keep this function call here
console.log(FirstReverse(readline()));